lib/services/ios-debug-service.ts

```diff
@@ -103,11 +103,18 @@ class IOSDebugService extends DebugServiceBase implements IPlatformDebugService
 
 		let lineStream = byline(child_process.stdout);
 		this._childProcess = child_process;
+		const pidRegExp = new RegExp(`${debugData.applicationIdentifier}:\\s?(\\d+)`);
```

Comment:
Would be nice to have Unit Tests for this Regular Expression. I'd suggest we extract this to some method/service so that we can test it and cover with more expressions?